schema/index.jsonld

```diff
@@ -16,7 +16,7 @@
         },
         "groups": {
             "@id": "contao:groups",
-            "@type": "xsd:decimal"
+            "@type": "xsd:integer"
```

Comment:
that is actually an array of integers. so both seem wrong to me?

Reply:
That is what I though as well, however, the array is already taken from the JSON type. Internally, the definition is converted to `<xsd:list itemType="xsd:interger"/>`.

Reply:
I think that's why this originally was `list`, but that was refused when we tested the schema file 🤷